data/psalter/psalms.yaml

```diff
@@ -62,7 +62,7 @@ psalms:
         lift up her eyes to that great pattern of perfection, Jesus, the man
         of God's right hand, who was holy, harmless, undefiled, and separate
         from sinners; and who fulfilled all righteousness, magnified the law,
-        and made it honourable for men ­ for me. United to his person, clothed
+        and made it honourable for men - for me. United to his person, clothed
```

Comment:
Neither is correct. It should read:

> for men---for *ME*

See the printed edition:

![2020-10-26-121900-screenshot](https://user-images.githubusercontent.com/206357/97191672-dfeef200-1785-11eb-9349-52774599dbaa.jpg)